website/src/data/members.js

```diff
@@ -176,6 +177,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/minhtnguy/',
         imageURL: minh,
       },
+      {
+        name: 'Kaitlin Leung',
+        position: 'Team Member',
```

Comment:
You could change this to developer! You are a full-stack developer with us after all :D